docs/proxysql-conf.md

```diff
@@ -137,201 +108,221 @@ proxysql:
+* The scheduler automatically manages the primary node, ensuring only one primary exists at a time
 
-!!! note
+As a result, you achieve:
 
-    Do not forget to restart Percona XtraDB Cluster to ensure the
-    cluster has updated the configuration.
+* Better performance through faster query processing and increased throughput
+* Higher reliability by preventing single-node bottlenecks and points of failure
+* Healthier cluster through early detection of replication lag and node issues
+* Efficient resource utilization
+* Improved user experience with consistent, predictable response times
 
-## Accessing the ProxySQL Admin Interface
+### Enable the scheduler
 
-You can use [ProxySQL admin interface :octicons-link-external-16:](https://www.percona.com/blog/2017/06/07/proxysql-admin-interface-not-typical-mysql-server/) to  configure its settings.
+The scheduler is disabled by default to maintain backward compatibility. You can enable it by setting `proxysql.scheduler.enabled=true` in your Custom Resource.
 
-Configuring ProxySQL in this way means connecting to it using the MySQL
-protocol, and two things are needed to do it:
+1. Edit the `deploy/cr.yaml` file and add the scheduler configuration:
 
-* the ProxySQL Pod name
+    ```yaml
+    proxysql:
+      enabled: true
+      size: 3
+      image: percona/percona-xtradb-cluster-operator:{{ release }}-proxysql
+      scheduler:
+        enabled: true
+    ```
+
+2. Apply the configuration:
 
-* the ProxySQL admin password
+    ```{.bash data-prompt="$" }
+    $ kubectl apply -f deploy/cr.yaml -n <namespace>
+    ```
 
-You can find out ProxySQL Pod name with the `kubectl get pods` command,
-which will have the following output:
+When the scheduler is enabled, you should see:
 
-```default
-$ kubectl get pods
-NAME                                              READY   STATUS    RESTARTS   AGE
-cluster1-pxc-node-0                               1/1     Running   0          5m
-cluster1-pxc-node-1                               1/1     Running   0          4m
-cluster1-pxc-node-2                               1/1     Running   0          2m
-cluster1-proxysql-0                               1/1     Running   0          5m
-percona-xtradb-cluster-operator-dc67778fd-qtspz   1/1     Running   0          6m
-```
+* **Hostgroup 10** (readers): All PXC nodes with equal or weighted distribution
+* **Hostgroup 11** (writer): Only the current writer node (typically `pod-0`) with a high weight (1000000)
 
-The next command will print you the needed admin password:
+You can also test read load balancing by running multiple `SELECT` queries and checking which node they hit:
 
-```default
-$ kubectl get secrets $(kubectl get pxc -o jsonpath='{.items[].spec.secretsName}') -o template='{{'{{'}} .data.proxyadmin | base64decode {{'}}'}}'
+```{.bash data-prompt="$" }
+$ for i in $(seq 100); do 
+    kubectl exec -i cluster1-pxc-0 -c pxc -- mysql -uroot -proot_password \
```

Comment:
I am not sure we should provide this command here. 
But at least we should add to instrucation that password should be updated to the one used by cluster. This one `-proot_password` is some test one. Or mention that for examople if cluster pwd is  `root_password`, then command is blahblahblah.